pkg/util/utils.go

```diff
@@ -32,3 +36,13 @@ func RetryWithExponentialBackOff(fn wait.ConditionFunc, duration time.Duration,
 	}
 	return wait.ExponentialBackoff(backoff, fn)
 }
+
+func SetupLogging(uuid string) {
+	logFileName := fmt.Sprintf("logfile_%s.log", uuid)
```

Comment:
Let's call prefix it with kube-burner better? sounds more intuitive

```suggestion
	logFileName := fmt.Sprintf("kube-burner-%s.log", uuid)
```

Reply:
Yep suits more to our tool.